org.eclipse.winery.model/org.eclipse.winery.model.tosca.xml/pom.xml

```diff
@@ -72,6 +72,12 @@
             <version>${org.slf4j}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.winery</groupId>
+            <artifactId>org.eclipse.winery.model.tosca.canonical</artifactId>
```

Comment:
Why do you add a dependency to the canonical model in the xml model? The XML model must not be aware of the wrapping model...